atom/browser/autofill/atom_autofill_client.cc

```diff
@@ -196,7 +196,7 @@ void AtomAutofillClient::UpdateAutofillPopupDataListValues(
 }
 
 void AtomAutofillClient::HideAutofillPopup() {
-  if (api_web_contents_) {
+  if (api_web_contents_ && api_web_contents_->IsFocused()) {
```

Comment:
addressed in https://github.com/brave/browser-laptop/commit/396cdbcaebc480e488cd3af915a2bb21552b1c2b
and we will keep `api_web_contents_->IsFocused()` for dismiss popup when scrolling page